content/en/docs/instrumentation/java/exporters.md

```diff
@@ -0,0 +1,141 @@
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.exporter.otlp.logs.OtlpGrpcLogRecordExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.sdk.logs.export.BatchLogRecordProcessor;
+import io.opentelemetry.sdk.logs.SdkLoggerProvider;
+import io.opentelemetry.sdk.logs.export.LogRecordExporter;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+
+@SpringBootApplication
+public class DiceApplication {
+  public static void main(String[] args) {
+    SpringApplication app = new SpringApplication(DiceApplication.class);
+    app.setBannerMode(Banner.Mode.OFF);
+    app.run(args);
+  }
+
+  @Bean
+  public OpenTelemetry openTelemetry() {
+    Resource resource = Resource.getDefault().toBuilder().put(SERVICE_NAME, "dice-server").put(SERVICE_VERSION, "0.1.0").build();
+
+    SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder()
+            .addSpanProcessor(BatchSpanProcessor.builder(OtlpGrpcSpanExporter.builder().build()).build())
+            .setResource(resource)
+            .build();
+
+    SdkMeterProvider sdkMeterProvider = SdkMeterProvider.builder()
+            .registerMetricReader(PeriodicMetricReader.builder(OtlpGrpcMetricExporter.builder().build()).build())
+            .setResource(resource)
+            .build();
+
+    SdkLoggerProvider sdkLoggerProvider = SdkLoggerProvider.builder()
+            .addLogRecordProcessor(
+                    BatchLogRecordProcessor.builder(OtlpGrpcLogRecordExporter.builder().build()).build())
+            .setResource(resource)
+            .build();
+
+    OpenTelemetry openTelemetry = OpenTelemetrySdk.builder()
+        .setTracerProvider(sdkTracerProvider)
+        .setMeterProvider(sdkMeterProvider)
+        .setLoggerProvider(sdkLoggerProvider)
+        .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
+        .buildAndRegisterGlobal();
+
+    return openTelemetry;
+  }
+}
+```
+
+To try out the exporters quickly, you can run Jaeger with OTLP enabled in a
+docker container:
+
+```shell
+docker run -d --name jaeger \
+  -e COLLECTOR_OTLP_ENABLED=true \
+  -p 16686:16686 \
+  -p 4317:4317 \
+  -p 4318:4318 \
+  jaegertracing/all-in-one:latest
```

Comment:
Jaeger isn't going to accept the metrics or logs that the SDK is configured to send to it.

Reply:
That's true, this is actually a bigger issue since we have the same setup for other languages as well. 

As a short fix let me change language to call out that this will only show you traces.

Additionally I'll raise an issue to discuss how we want to setup those Exporter pages to have something for all signals.

Reply:
[Not perfect, but accurate for now](https://github.com/open-telemetry/opentelemetry.io/pull/3144/commits/839ab5e3172202f6525347cdcbbf211cf7cae0cb#diff-b4c7cabc3b96c8b722a2bbbf1c2f26c2b27b2c3f86cba61a49be91ca06c034b3R131)

Reply:
follow-up issue #3266